src/index.js

```diff
@@ -5,9 +5,19 @@ import './wdyr';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+
+Sentry.init({
+  release: process.env.npm_package_version,
+  dsn: process.env.SENTRY_DSN,
+  environment: process.env.SENTRY_ENVIRONMENT,
+  integrations: [new Integrations.BrowserTracing()],
+});
```

Comment:
Maybe you can wrap sentry.init by a condition verifying the existence of `process.env.SENTRY_DSN` and in the same way for `environment` like following 

```suggestion
if(process.env.SENTRY_DSN !== undefined) {
  Sentry.init({
    release: process.env.npm_package_version,
    dsn: process.env.SENTRY_DSN,
    ...(process.env.SENTRY_ENVIRONMENT && { environment: process.env.SENTRY_ENVIRONMENT}),
    integrations: [new Integrations.BrowserTracing()],
  });
}
```

NB: Not sure if the second one is relevant; I think sentry must manage the environment if it was set to undefined